docs/spec/api.md

```diff
@@ -1,3 +1,7 @@
+page_title: Docker Registry HTTP API V2
```

Comment:
We can guard these with takes like this:

```
<!-- REMOVE
foo foo foo
REMOVE -->
```

This will turn the header into comments. We can remove them with grep:

```console
grep -vE '^<!-- REMOVE$|^REMOVE -->$' foo.md
```